src/test/java/uk/gov/digital/ho/proving/income/audit/AuditClientTest.java

```diff
@@ -106,4 +114,20 @@ public void shouldSetAuditableData() {
         assertThat(auditableData.getEventType()).isEqualTo(INCOME_PROVING_FINANCIAL_STATUS_REQUEST);
         assertThat(auditableData.getData()).isEqualTo("{}");
     }
+
```

Comment:
There's other log points in this class and we should test all of them if possible